.env.sample

```
@@ -0,0 +1,14 @@
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/db_name"
+
+PORT=yourPort
+
+DB_TYPE=mysql
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_USERNAME=root
```

해당 부분도 실제 사용되는 값이 아닌, 값을 유추할 수 있는 문자열로 대체해주면 좋을 것 같습니다.

```suggestion
DB_HOST=hostNumber
DB_PORT=portNumber
DB_USERNAME=username
```